jenkins/shared-library/vars/createMonorepoMultibranchJobs.groovy

```diff
@@ -40,14 +40,13 @@ void generatePackageMultibranchJobs(Object buildConfiguration, Map jobConfigurat
 
 	buildConfiguration.builds.each { build ->
 		String pipelineName = build.path.tokenize(pathSeparator).last()
-		jobConfiguration.with {
-			jobName = Paths.get(jobConfiguration.fullBranchFolder.toString()).resolve(pipelineName).toString()
-			jenkinsfilePath = Paths.get(build.path).resolve('Jenkinsfile').toString()
-			packageIncludePaths = addPathAndDependsOnFolder(build).join('\n')
-			packageExcludePaths = '**/*'
-			path = build.path
-			packageFolder = Paths.get(jobConfiguration.repositoryName.toString()).resolve(build.path).toString()
-		}
+		jobConfiguration.jobName = Paths.get(jobConfiguration.fullBranchFolder.toString()).resolve(pipelineName).toString()
```

Comment:
double check `with` behavior as discussed in Discord. at minimum, you'll need need to suppress warning(s).

Reply:
I am incline to suppress unless you have any other ideas.
I took your example code and ran it in Jenkins script console and it works.  In the pipeline, there might be a bug or I am missing something.   The values that gets assign in the ``with`` statement dont exist after the ``with`` block.

```
		println "Hello World!"

		def jobConfiguration = [:]
		jobConfiguration.alpha = 'hello'
		jobConfiguration.zeta = 'dear'

		jobConfiguration.with {
			println('start with')
			beta = 'foo'
			zeta = 'bar'
			println("end with beta: ${beta}")
		};

		println("alpha: ${jobConfiguration.alpha}")
		println("beta: ${jobConfiguration.beta}")
		println("zeta: ${jobConfiguration.zeta}")
	``` 

The out in the pipeline is below
```
[Pipeline] echo
10:43:16  Hello World!
[Pipeline] echo
10:43:16  start with
[Pipeline] echo
10:43:16  end with beta: foo
[Pipeline] echo
10:43:16  alpha: hello
[Pipeline] echo
10:43:16  beta: null
[Pipeline] echo
10:43:16  zeta: dear
```

Reply:
can you try something like this?
```
println "Hello World!"

def jobConfiguration = [:]
jobConfiguration.alpha = 'hello'
jobConfiguration.zeta = 'dear'

def newJobConfigurationProperties = [:]
newJobConfigurationProperties.with{
	println('start with')
	beta = 'foo'
	zeta = 'bar'
	println("end with beta: ${beta}")
};

jobConfiguration += newJobConfigurationProperties

println("alpha: ${jobConfiguration.alpha}")
println("beta: ${jobConfiguration.beta}")
println("zeta: ${jobConfiguration.zeta}")
```

Reply:
I tried several different variation but still no luck
```
[Pipeline] echo
11:31:31  Hello World!
[Pipeline] echo
11:31:31  start with
[Pipeline] echo
11:31:31  end with beta: foo
[Pipeline] echo
11:31:31  newJobConfigurationProperties: [:]
[Pipeline] echo
11:31:31  alpha: hello
[Pipeline] echo
11:31:31  beta: null
[Pipeline] echo
11:31:31  zeta: dear
```

Reply:
:( can you try replacing `with` with `identity` ?

Reply:
seems like known issue: https://stackoverflow.com/questions/57561261/why-does-with-method-doesnt-work-in-jenkins-pipeline
😢